http.go

```diff
@@ -207,41 +263,51 @@ func NewReverseProxy(config *Config, db *sql.DB) *httputil.ReverseProxy {
 
 		var resBody []byte
 		if res.Body != nil {
```

Comment:
Should we check also for 2xx status code before attempting to deserialize the json body and modify the response?

Reply:
Applied in [1b17c5f](https://github.com/breez/swapproxy/pull/6/commits/1b17c5fa2b3658bc1c0fde227b2fbc9912fd097c)